CUJExecutor/manipulateProfilesAndTest

```diff
@@ -15,9 +15,20 @@
 #limitations under the License.
 
 
-##compiles the app according to the COMPILATION_FLAG and then measures the cuj specified by the INPUT_FILE 
-##stores a command/output summary, a short data summary, the profile used to compile the app (if applicable) 
-##and a recording of the median run 
+##
+# compiles aGMM according to the COMPILATION_FLAG and then measures the CUJ specified by the INPUT_FILE (as specified by executeCUJ).
+# Stores the following files at /profile_test_data/<INPUT_FILE_NAME>/<COMPILATION_FLAG>/<DATE>/:
+# - (command_summary.txt) a log of each run command and their output 
+# - (short_summary.txt) a formatted summary including the profile used for compilation, resulting app size and measurements of the POST_CUJ (acquired via a call to executeCUJ). A copy is stored in /tmp for the callee.
+# - (saved_profile.prof) the profile used to compile the app (if applicable) 
+# - (median_clip.mp4) and a recording of the median iteration (if requested)
```

Comment:
Delete "and":

    (median_clip.mp4) a recording of the median iteration (if requested)